src/amo/components/SearchResult/styles.scss

```diff
@@ -162,8 +161,8 @@ $icon-default-size: 32px;
     font-weight: normal;
     margin: 0;
     max-width: 465px;
+    overflow-wrap: break-word;
```

Comment:
Is this really needed here?

Reply:
It was inside a `respond-to` mixin, so I thought not to remove this. Just changed from `word-wrap` to `overflow-wrap`.